src/Destination.js

```diff
@@ -15,9 +15,8 @@ class Destination {
     return destinationNames;
   }
 
-  calculateTripExpense(duration, numTravelers, id) {
-      let userTrip = this.data.filter(destination => destination.id === id);
-      console.log(userTrip)
+  calculateTripExpense(duration, numTravelers, destId) {
+      let userTrip = this.data.filter(destination => destination.id === destId);
```

Comment:
I like that you changed the parameter from "id" to "destId" to make it easier to understand!